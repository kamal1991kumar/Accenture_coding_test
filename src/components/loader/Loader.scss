.loader {
  display: block;
    text-align: center;
    transform: scale(0.5);
    opacity: .5;
}