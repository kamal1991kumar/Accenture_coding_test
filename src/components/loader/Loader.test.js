import React from "react";
import { render } from "@testing-library/react";
import Loader from "./index";

describe("<Loader />", () => {
  test("Should render on screen", () => {
    const { container } = render(<Loader />);
    expect(container).toMatchSnapshot();
  });
});
