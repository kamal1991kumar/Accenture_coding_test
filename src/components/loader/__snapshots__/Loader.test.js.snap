// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> Should render on screen 1`] = `
<div>
  <div
    class="loader"
  >
    <svg
      height="48px"
      preserveAspectRatio="xMidYMid"
      viewBox="0 0 100 100"
      width="48px"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        transform="rotate(0 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.9565217391304348s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(15.652173913043478 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.9130434782608695s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(31.304347826086957 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.8695652173913043s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(46.95652173913044 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.8260869565217391s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(62.608695652173914 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.782608695652174s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(78.26086956521739 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.7391304347826086s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(93.91304347826087 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.6956521739130435s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(109.56521739130434 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.6521739130434783s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(125.21739130434783 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.6086956521739131s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(140.8695652173913 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.5652173913043478s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(156.52173913043478 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.5217391304347826s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(172.17391304347825 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.4782608695652174s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(187.82608695652175 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.43478260869565216s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(203.47826086956522 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.391304347826087s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(219.1304347826087 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.34782608695652173s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(234.7826086956522 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.30434782608695654s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(250.43478260869566 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.2608695652173913s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(266.0869565217391 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.21739130434782608s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(281.7391304347826 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.17391304347826086s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(297.39130434782606 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.13043478260869565s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(313.04347826086956 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.08695652173913043s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(328.69565217391306 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="-0.043478260869565216s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
      <g
        transform="rotate(344.3478260869565 50 50)"
      >
        <rect
          fill="#231c1b"
          height="63"
          rx="3"
          ry="5.67"
          width="6"
          x="47"
          y="-4.5"
        >
          <animate
            attributeName="opacity"
            begin="0s"
            dur="1s"
            keyTimes="0;1"
            repeatCount="indefinite"
            values="1;0"
          />
        </rect>
      </g>
    </svg>
  </div>
</div>
`;
